lib/wallaby/experimental/chrome.ex

```diff
@@ -0,0 +1,178 @@
+  def accept_prompt(_session, _input, _fun), do: {:error, :not_implemented}
+  def dismiss_prompt(_session, _fun), do: {:error, :not_implemented}
+  @doc false
+  defdelegate cookies(session),                                   to: WebdriverClient
+  @doc false
+  defdelegate current_path(session),                             to: WebdriverClient
+  @doc false
+  defdelegate current_url(session),                              to: WebdriverClient
+  @doc false
+  defdelegate page_title(session),                                to: WebdriverClient
+  @doc false
+  defdelegate page_source(session),                               to: WebdriverClient
+  @doc false
+  defdelegate set_cookie(session, key, value),                   to: WebdriverClient
+  @doc false
+  defdelegate visit(session, url),                                to: WebdriverClient
+
+  @doc false
+  defdelegate attribute(element, name),                           to: WebdriverClient
+  @doc false
+  defdelegate click(element),                                     to: WebdriverClient
+  @doc false
+  defdelegate clear(element),                                     to: WebdriverClient
+  @doc false
+  defdelegate displayed(element),                                 to: WebdriverClient
+  @doc false
+  defdelegate selected(element),                                  to: WebdriverClient
+  @doc false
+  defdelegate set_value(element, value),                          to: WebdriverClient
+  @doc false
+  defdelegate text(element),                                      to: WebdriverClient
+
+  @doc false
+  defdelegate execute_script(session_or_element, script, args),   to: WebdriverClient
+  @doc false
+  defdelegate find_elements(session_or_element, compiled_query),  to: WebdriverClient
+  @doc false
+  defdelegate send_keys(session_or_element, keys),                to: WebdriverClient
+  @doc false
+  defdelegate take_screenshot(session_or_element),                to: WebdriverClient
+
+  @doc false
+  def user_agent do
+    "Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36"
+  end
+
+  defp default_capabilities do
+    %{
+      javascriptEnabled: false,
+      loadImages: false,
+      version: "",
+      rotatable: false,
+      takesScreenshot: true,
+      cssSelectorsEnabled: true,
+      nativeEvents: false,
+      platform: "ANY",
+      chromeOptions: %{
+        args: [
+          "--no-sandbox",
+          "window-size=1280,800",
+          "--headless",
+          "--disable-gpu"
+        ]
+      }
+    }
+  end
+
+  defp poolboy_config(), do: [
+    name: {:local, @pool_name},
+    worker_module: Wallaby.Experimental.Chrome.Chromedriver,
+    size: pool_size(),
+    max_overflow: 0
+  ]
+
+  defp pool_size, do: Application.get_env(:wallaby, :pool_size) || default_pool_size()
+
+  # defp default_pool_size, do: :erlang.system_info(:schedulers_online)
+  defp default_pool_size, do: 1
```

Comment:
begone commented out code! Plus we may need a comment here why this is as it is, the race condition or something, was it?